samples/nrf9160/cloud_client/src/main.c

```diff
@@ -135,6 +164,9 @@ void main(void)
 
 	printk("Cloud client has started\n");
 
+	// err = lte_lc_register_handler(lte_handler);
+	// __ASSERT(err == 0, "LTE link controller event handler not registered");
+
```

Comment:
remember clean up

Reply:
This commit with changes to cloud client was just intended to show how to use the link controller handler, but if we decide to add it permanently, I'll clean it up

Reply:
Decided to keep the commit with cloud client changes to show how the new events can be used, and I've cleaned up the commit.